react-jsx/react-jsx/react-jsx-quiz-notes.md

```diff
@@ -0,0 +1,38 @@
+# react-jsx-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is JSX?
+  JSX is a syntax extensions for JavaScript that lets you write HTML-like markup inside a JavaScript file.
+- How does React use JSX to render components?
+  By using curly braces to open a sort of window into JavaScript from JSX.
```

Comment:
JSX Elements are used to make DOM calls that create, modify, and delete DOM elements
A React component returns a JSX Element which represents how the DOM should be structured